docs/maps/maps-getting-started.asciidoc

```diff
@@ -119,9 +124,11 @@ The layer is only visible when users zoom in the map past zoom level 9.
 . Click the *Documents* data source.
 . Set *Index pattern* to *kibana_sample_data_logs*.
 . Click the *Add layer* button.
-. Set *Layer name* to `Actual Requests`.
-. Set *Zoom range for layer visibility* to the range [9, 24].
-. Set *Layer transparency* to 1.
+. Set *Name* to `Actual Requests`.
+. Set *Visibilty* to the range [9, 24].
+. Set *Opacity* to 100%.
+. Click *Add* under *Tooltip fields*.
+. In the popover, select *clientip*, *timestamp*, *host*, *request*, *response*, *machine.os*, *agent*, and *bytes* and click *Add*.
```

Comment:
Does this make the list easier to read?

In the popover, select the following fields and then click *Add*.

- **clientip**
- **timestamp**
- **host**
- **request**
- **response**
- **machine_os**
- **agent**
- **bytes**

Reply:
I think I will leave these as is without breaking out into a bulleted list. They are not that important. just showing how to add fields to a tooltip. I think a bulleted list would draw too much attention.